test/components/modeRanking.spec.js

```diff
@@ -0,0 +1,77 @@
+import { ModeRanking } from '../../src/app/components/ModeRanking';
+import { screen } from '@testing-library/dom';
+import '@testing-library/jest-dom';
+import { renderComponent } from '../renderComponent';
+
+describe('Players ranking', () => {
+  const sampleRanking = [
+    {
+      user: 'user1',
+      score: 15,
+      maxScore: 30,
+    },
+    {
+      user: 'user2',
+      score: 18,
+      maxScore: 40,
+    },
+    {
+      user: 'user3',
+      score: 15,
+      maxScore: 25,
+    },
+  ];
+
+  const emptyRanking = [];
+
+  const rankingWithTwoUsers = [
+    {
+      user: 'user1',
+      score: 15,
+      maxScore: 30,
+    },
+    {
+      user: 'user2',
+      score: 18,
+      maxScore: 40,
+    },
+  ];
+
+  it("Given an empty array should render component with paragraph 'Leaderboard is empty...' text inside and not contain any player info row", () => {
+    renderComponent(ModeRanking(emptyRanking));
+
+    expect(screen.getByText('Leaderboard is empty...')).toBeInTheDocument();
+    expect(screen.queryByTestId('playerRowNum1')).toBeFalsy();
```

Comment:
Dobrze napisane testy! :) 

toBeFalsy ma kilka możliwośći, a zdaje mi się, żę bardziej czytelnie jest dać po prostu, że go nie ma:

`expect(screen.queryByTestId('playerRowNum1')).not.toBeInTheDocument()`

Reply:
racja, zamienię na notToBeInTheDocument() :P